teams/Aragon One/2019.md

```diff
@@ -277,7 +274,7 @@ The team is fully remote and distributed across the world, primarily in Europe a
 
 ### Current team
 
-The current full-time team is comprised of 17 people, distributed as follows:
+The current full-time team is comprised of 18 people, distributed as follows:
```

Comment:
```suggestion
The current full-time team is comprised of 17 people, distributed as follows:
```

Reply:
Two team members left, and two joined, so we still have 17 people :)

Reply:
Thanks, fixed!